# sets the robot's active degrees of freedom

int32 robotid
float32[] values

# indices into the active degree of freedom table
uint8[] indices
---
